apis/placement/v1alpha1/override_types.go

```diff
@@ -67,7 +68,9 @@ type OverridePolicy struct {
 type OverrideRule struct {
 	// ClusterSelectors selects the target clusters.
 	// The resources will be overridden before applying to the matching clusters.
-	// If ClusterSelector is not set, it means selecting ALL the member clusters.
+	// An empty clusterSelector selects ALL the member clusters.
+	// A null clusterSelector selects NO member clusters.
+	// For now, only labelSelector is supported.
 	// +optional
 	ClusterSelector *placementv1beta1.ClusterSelector `json:"clusterSelector,omitempty"`
```

Comment:
@ryanzhang-oss i did not mark this field as required. cx may want to set the null temporarily to ignore this rule or even override for testing purpose.